proof/crefine/RISCV64/TcbQueue_C.thy

```diff
@@ -1424,11 +1428,13 @@ lemma tcb_queue_relation_not_in_q:
     = tcb_queue_relation' nxtFn prvFn hp xs start end"
   by (rule tcb_queue_relation'_cong, auto)
 
+(* FIXME RT: unused? This might require an additional assumption that thread is not
+   the idle thread, and some further reasoning involving valid_idle' *)
 lemma rf_sr_tcb_update_not_in_queue:
```

Comment:
It is indeed entirely unused. I would vote for removing it (and the derived `..2` variant)